js/securesubmit_plugin.js

```diff
@@ -32,7 +32,7 @@
 
     jQuery(function(){
         var scriptpath = jQuery("script[src]").last().attr("src").split('?')[0].split('/').slice(0, -5).join('/')+'/';
-        scriptpath += 'wp-content/plugins/securesubmit/template/index.html';
+        scriptpath += 'wp-content/plugins/donate/template/index.html';
```

Comment:
This should be reverted at the very least to not break existing installations, but ideally, this should be updated to create this address on the PHP side via WordPress's API around discovering plugin URLs.